### DISCUSSION TITLE
Disabling autocompletion of brackets, quotes, etc...

### DISCUSSION THREAD

**Discussion by JamesTiberiusKirk** [LazyVim/LazyVim#528] [General]
Hi, sorry if this is too simple but I just cant quite figure out how to disable the automacic completion of different brakets quotes and the sort.
Thanks in advance.

**Comment by dpetka2001** [comment]
Do you mean the auto completion of `""` for example? So when you type quotes it doesn't automatically insert a pair of quotes? If that so, then I believe the plugin `mini.pairs` is the one responsible. If you disable it according to the [docs](https://www.lazyvim.org/configuration/plugins#-disabling-plugins), I guess you should be good to go.

**Reply by JamesTiberiusKirk** [reply]
Thanks!
